pkg/registry/flowcontrol/bootstrap/default-objects.go

```diff
@@ -0,0 +1,270 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bootstrap
+
+import (
+	rmv1a1 "k8s.io/api/flowcontrol/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/authentication/user"
+)
+
+// all the preserving objects
```

Comment:
What does preserving mean in this context?

Reply:
I think it is a holdover from an earlier implementation idea --- keeping  certain predefined entries in the `requestManagementState` even if no actually existing API object calls for those entries.  I think @yue9944882's approach has changed and now what he describes as "preserving" are the entries put in the `requestManagementState` at apiserver startup (so that it has useful behavior in the interim until it has had a chance to read/create the actually existing API objects) but they are not kept around past a certain point of startup completion (namely config informers synced AND sufficient config API objects actually exist).  In my opinion "initial" would be a better word to convey this meaning.